validator/test/fixtures/assets/cloudsql/data.json

```diff
@@ -0,0 +1,185 @@
+[{
+  "name": "//sqladmin.googleapis.com/my-cloudsql",
+  "asset_type": "sqladmin.googleapis.com/Instance",
+  "resource": {
+    "settings": {
+      "ipConfiguration": {
+        "requireSsl": false
+      }
+    }
+  }
+},
+{
+  "name": "//sqladmin.googleapis.com/my-cloudsql2",
+  "asset_type": "sqladmin.googleapis.com/Instance",
+  "resource": {
+    "settings": {
+      "ipConfiguration": {
+        "requireSsl": true
+      }
+    }
+  }
+},
+{
+  "name": "//cloudsql.googleapis.com/projects/test1",
+  "asset_type": "sqladmin.googleapis.com/Instance",
+  "resource": {
+      "settings": {
+        "activationPolicy": "ALWAYS",
+        "backupConfiguration": {
+          "binaryLogEnabled": true,
+          "enabled": true,
+          "kind": "sql#backupConfiguration",
+          "startTime": "22:00"
+        },
+        "dataDiskSizeGb": "25",
+        "dataDiskType": "PD_SSD",
+        "ipConfiguration": {
+          "ipv4Enabled": true,
+          "requireSsl": true
+        },
+        "kind": "sql#settings",
+        "locationPreference": {
+          "kind": "sql#locationPreference",
+          "zone": "us-central1-b"
+        },
+        "pricingPlan": "PER_USE",
+        "replicationType": "SYNCHRONOUS",
+        "settingsVersion": "28",
+        "storageAutoResize": true,
+        "storageAutoResizeLimit": "0",
+        "tier": "db-n1-standard-1"
+      },
+      "state": "RUNNABLE"
+    }
+  },
+{
+  "name": "//cloudsql.googleapis.com/projects/noble-history-87417/instances/galu",
```

Comment:
nit: can we use a more descriptive name to indicate whether this is a "good" or "bad" instance?

Reply:
similarly below...

Reply:
Done.